OpenStack-Rabbit-Consumer/consumer.ini

```diff
@@ -1,2 +1,2 @@
 [rabbit]
-exchanges=nova
+exchanges=
```

Comment:
Why is `exchanges=` ?

Reply:
This should have been removed instead of just the text :man_facepalming: